Command/CreateTweetContentCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * File containing the CreateTweetContentType class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\TweetFieldTypeBundle\Command;
```

Comment:
Yes, the README should contain some information about the non-guarantee for tags stability.
What will happen when you do, as a reader of the Tutorial a `git pull` and the tag has changed? It can be surprising.